### PR TITLE
Add CORS note for amp-call-tracking docs

### PR DESCRIPTION
**Pull request by alanorozco**
Based on the docs for [amp-form](/ampproject/amphtml/blob/master/extensions/amp-form/amp-form.md)

Ref #8446

### CODE REVIEW

**Comment by ericlindley-g** [comment]
Passing to @rudygalfi , who worked more on the initial amp-form launch

FYI: this is what I'm seeing in the actual page:

screen shot 2017-04-06 at 2 24 46 pm

Should it look like that, or is the markup not getting applied?

**Comment by alanorozco** [comment]
@ericlindley-g 

The markup looks like that on Github (see [amp-form](https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/amp-form.md)), but it will be formatted on ampproject.org (see [amp-form](https://www.ampproject.org/docs/reference/components/dynamic/amp-form))

**Comment by ericlindley-g** [comment]
Makes sense — thanks for explaining! (though makes me wish we had a way for both to look good :)

File: extensions/amp-call-tracking/amp-call-tracking.md [hunk 1]
```diff
@@ -61,6 +61,10 @@ of a valid JSON object with the following fields:
 - `phoneNumber` (required): Specifies the phone number to call when the user clicks the link. 
 - `formattedPhoneNumber` (optional): Specifies the phone number to display. If not specified, the value in `phoneNumber` is used.
 
+{% call callout('Important', type='caution') %}
+Your XHR endpoint needs to follow and implement [CORS Requests in AMP spec](../../spec/amp-cors-requests.md).
```
Comment (rudygalfi) [inline extensions/amp-call-tracking/amp-call-tracking.md]: ... and implement *the* CORS ...

**Pull request merged**